package.json

```diff
@@ -5,9 +5,9 @@
   "main": "index.js",
   "scripts": {
     "lint": "eslint ./ --fix",
-    "migrate": "sequelize db:migrate:undo:all && sequelize db:migrate",
+    "migrate": "sequelize db:migrate:undo:all && sequelize db:migrate && sequelize db:seed:all",
```

Comment:
There is a seed script already in line `13`. I think you can run them separately instead of having them both in the `migrate` script.